ide/project.dependency/src/org/netbeans/modules/project/dependency/spi/ProjectDependenciesImplementation.java

```diff
@@ -32,16 +29,9 @@
  * @author sdedic
  */
 public interface ProjectDependenciesImplementation {
```

Comment:
Removing methods from `spi` package? Isn't that an API change?

Reply:
This "API" was never published, even as a friend.